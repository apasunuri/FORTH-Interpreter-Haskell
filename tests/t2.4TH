hello 4 5.6 4.908 3 5 ^ 