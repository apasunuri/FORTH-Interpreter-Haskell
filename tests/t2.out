"Stack is not Empty"
[Real 243.0,Real 4.908,Real 5.6,Integer 4,Id "hello"]